In [1]:
#Consider the string given below and answer the following questions 
string = "OQYWFClFhFGAvIWYwGKpmZhnJiyzTslSIhSwvOsqJMEphzmifTkyqOMNpnOtXZxmCfgDYqbaBHAUvIWhMnvwZnEMVDvmEfLrDoQnAZgQEgXQVnmSYkfedpAdhrtpOgORpYLRZYGWdhWYuqQssCUXtTzKRDAhpjUheOzUroZNzWFtZOVwIapzUYtbSbjYNErzQ"
len(string)

193

####  Calculate the number of characters in the string

-  193
-  200
-  300
-  500


In [4]:
len(string)

193

#### Count the number of occurences of 'a' in the string

-  1
-  2
-  10
-  5

In [13]:
string = "OQYWFClFhFGAvIWYwGKpmZhnJiyzTslSIhSwvOsqJMEphzmifTkyqOMNpnOtXZxmCfgDYqbaBHAUvIWhMnvwZnEMVDvmEfLrDoQnAZgQEgXQVnmSYkfedpAdhrtpOgORpYLRZYGWdhWYuqQssCUXtTzKRDAhpjUheOzUroZNzWFtZOVwIapzUYtbSbjYNErzQ"

In [17]:
print(string)

OQYWFClFhFGAvIWYwGKpmZhnJiyzTslSIhSwvOsqJMEphzmifTkyqOMNpnOtXZxmCfgDYqbaBHAUvIWhMnvwZnEMVDvmEfLrDoQnAZgQEgXQVnmSYkfedpAdhrtpOgORpYLRZYGWdhWYuqQssCUXtTzKRDAhpjUheOzUroZNzWFtZOVwIapzUYtbSbjYNErzQ


In [18]:
print(string.count("a"))

2


#### Identify whether the string starts with 'if'.

-  True
-  False

In [21]:
type(string)

str

#### Return the substring starting at index 63 and ending at index 88  [Both the indexes are inclusive]

-  'OQYWFClFhFGA'
-  'mCfgDYqbaBHAUvIWhMnvwZnEMV'
-  'IWYwGKpmZhnJiyzTslSIhSwv'
-  'JMEphzmifTkyqOMNpnOtXZ'

In [6]:
print("OQYWFClFhFGA" in string)

True


In [7]:
print('mCfgDYqbaBHAUvIWhMnvwZnEMV' in string)

True


In [8]:
print('IWYwGKpmZhnJiyzTslSIhSwv'in string)

True


In [9]:
print("JMEphzmifTkyqOMNpnOtXZ" in string)

True


#### Identify the character at index 45

-  a
-  b
-  z
-  x

In [20]:
strchr(a)

NameError: name 'strchr' is not defined